openmc/geometry.py

```diff
@@ -12,19 +12,24 @@ def reset_auto_ids():
 
 
 class Geometry(object):
+    """Geometry representing a collection of surfaces, cells, and universes.
```

Comment:
Perhaps you need to capitalize "Surfaces," "Cells," etc. for Sphinx to auto-link them in the docs? I don't know this, but I believe this is the case for Doxygen and wondered if it is also true for Sphinx.